### PR TITLE
Add Arbitrum Addresses, Remove Deprecated IT

### PR DESCRIPTION
**Pull request by ElliotFriedman**

### CODE REVIEW

File: contracts/test/integration/IntegrationTestVoltDeployment.t.sol [hunk 1]
```diff
@@ -190,25 +190,6 @@ contract IntegrationTestVoltDeployment is DSTest, StdLib {
...
-
-        assertEq(endingUserVoltBalance - startingUserVoltBalance, mintAmount);
-        assertEq(
-            (endingPCVDepositFeiBalance - startingPCVDepositFeiBalance),
-            (mintAmount - 1) /// goes down by 1 because of cToken pricing rounding down
-        );
-    }
-
     /// this test uses FEI as the underlying asset and hooks into a FEI PCV Deposit
     function testMintAfterPriceIncrease() public {
```
Comment (k-xo) [inline contracts/test/integration/IntegrationTestVoltDeployment.t.sol:194]: I think we can remove this test too, its the mint equivalent of the redeem test removed

**Review by k-xo — APPROVED** [review]
LGTM

File: contracts/test/integration/IntegrationTestVoltDeployment.t.sol (hunk 1 above)
Comment (k-xo) [reply contracts/test/integration/IntegrationTestVoltDeployment.t.sol:194]: the `testMintAfterPriceIncrease` here

Comment (ElliotFriedman) [reply contracts/test/integration/IntegrationTestVoltDeployment.t.sol:194]: removed this whole file in the role integration testing PR

**Pull request merged**